docs/devicemanager.apib

```diff
@@ -574,6 +574,8 @@ In this example, there is only one template (ID 1).
     + page_size: 20 (integer, optional)
     + page_num: 1 (integer, optional)
     + idsOnly: false (booelan, optional) - Return only the ids of all existing devices. Ignores any `page_size` and `page_num` configurations
+    + attr: (string, optional) - Return only devices that contain an attribute with given value.
```

Comment:
A new request/response pair with a sample for this is to be pushed to this PR, as well as the hooks required to test it.